### DISCUSSION TITLE
Issue with window and electron

### DISCUSSION THREAD

**Discussion by loctran016** [JamesIves/github-pages-deploy-action#772] [Q&A] [answered]
Hi, there. Thanks for creating such a wonderful project.
In my project, I ran into a problem when uploading it. I'm using electron on `window-latest`. The log can be found here on [my repo](https://github.com/Cokaps/GenZOC/runs/2859846690?check_suite_focus=true)

**Comment by JamesIves** [comment, marked as answer]
[This part](https://github.com/JamesIves/github-pages-deploy-action#operating-system-support-) of the README should help you out!

**Comment by Arman7717** [comment]
No no this is me
